scanner/src/main/kotlin/ScanResultsStorage.kt

```diff
@@ -103,6 +103,17 @@ abstract class ScanResultsStorage {
             properties["password"] = config.password
             properties["ApplicationName"] = "$ORT_NAME - $TOOL_NAME"
 
+            // There are two ways to disable SSL, one is to not set the "ssl" property at all, the other is to set the
+            // "sslmode" to "disabled". We chose to always set the "ssl" property to simplify the code. Note that
+            // according to the docs currently the value of the "ssl" property is ignored, but it is encouraged to  set
+            // it to "true" for compatibility with future versions.
+            // See: https://jdbc.postgresql.org/documentation/head/connect.html
+            properties["ssl"] = "true"
+            properties["sslmode"] = config.sslmode
```

Comment:
I'd still prefer if there was a semantic match between setting the `ssl` and `sslmode` properties and thus suggest to replace the above two lines with:

    properties["sslmode"] = config.sslmode
    if (config.sslmode != "disabled") {
        // The mere presence of it specifies a SSL connection. However, for compatibility with future versions, the value "true" is preferred.
        properties["ssl"] = "true"
    }

If you follow my suggestion the above comment can be shortened accordingly.

Reply:
Still approving after discussing offline.

Reply:
"ssl" is only used as fallback in case "sslmode" is not set, so I have removed it. See:
https://github.com/pgjdbc/pgjdbc/blob/5fbe04626c64b7c46a22a0cea108921305b14074/pgjdbc/src/main/java/org/postgresql/jdbc/SslMode.java#L65